src/main/pages/portable-workspaces/creating-factories.adoc

```diff
@@ -160,6 +171,5 @@ If you have projects in GitHub or GitLab, you can help your contributors to get
 [id="nest-steps"]
 == Nest Steps
```

Comment:
```suggestion
== Nest steps
```

Reply:
This looks like a typo to me -- was "Next steps" intended?